pkg/new-ui/v1beta1/backend.go

```diff
@@ -225,7 +344,14 @@ func (k *KatibUIHandler) AddTemplate(w http.ResponseWriter, r *http.Request) {
 	updatedConfigMapPath := data["updatedConfigMapPath"].(string)
 	updatedTemplateYaml := data["updatedTemplateYaml"].(string)
 
-	newTemplates, err := k.updateTrialTemplates(updatedConfigMapNamespace, updatedConfigMapName, "", updatedConfigMapPath, updatedTemplateYaml, ActionTypeAdd)
+	err = IsAuthorized(user, "create", updatedConfigMapNamespace, "configmaps", "", updatedConfigMapName, corev1.SchemeGroupVersion, k.sarClient)
```

Comment:
`configmaps` constant [can be used from K8s API](https://github.com/kubernetes/api/blob/master/core/v1/types.go#L6087).

Reply:
Done!